src/UI/WB.UI.Designer/Controllers/Api/Designer/ExpressionGenerationController.cs

```diff
@@ -54,6 +61,21 @@ public IActionResult GetAllClassesForLatestVersion(Guid id, int? version)
 
             return Ok(resultBuilder.ToString());
         }
+        
+        private Dictionary<Guid, LookupTableContent> GetLookupTables(QuestionnaireDocument questionnaire)

```

Comment:
It's a code duplication. Could you please produce QuestionnaireCodeGenerationPackage in one place with supplying QuestionnaireDocument to it?